statsmodels/tools/tools.py

```diff
@@ -383,23 +383,6 @@ def clean0(matrix):
     return np.array(np.transpose(val))
 
 
-def rank(X, cond=1.0e-12):
```

Comment:
there is no internal use of this anymore?
we might have silently changed the threshold cond at some point when switching to numpy default